src/components/pages/NewParentDashboard/RenderNewParentDashboard.js

```diff
@@ -10,14 +10,40 @@ import { connect } from 'react-redux';
 import { setParent } from '../../../state/actions/parentActions';
 import RenderWordCloud from '../WordCloud';
 import WordCountContainer from '../WordCountContainer/WordCountContainer';
+import ChooseChildModal from './ChooseChildModal';
+import { useHistory } from 'react-router-dom';
 
 const RenderNewParentDashboard = props => {
+  const FAKE_CHILDREN = [
+    {
+      ID: 0,
+      Name: 'Pinky Winky',
+      AvatarURL: 'https://scribble-stadium.s3.amazonaws.com/avatars/01.svg',
```

Comment:
This is a valid way to get the svg images through the URL.  Also you can import the exact svg hero image that matches the Figma if you go to assets / images / hero_images you can find the Figma images.  Since it is hard coded it might not matter though.

Reply:
This has been changed to update assets.